onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeSource.java

```diff
@@ -366,7 +375,21 @@ private FeedMessage readFeedFromUrl(URL url) throws IOException {
       }
     }
   }
-
+/**
+ * Set the headers to the urlConnection if any
+ * @param urlConnection
+ * @return, the urlConnection with the headers set
+ */
+  private void setHeadersToUrlConnection(URLConnection urlConnection)
+  {
+	  if(_headersMap!= null)
```

Comment:
Could you please add a space before the `!`?  In the future there is a style template for Eclipse that should help keep the format consistent.